src/controller/CHIPDeviceController.cpp

```diff
@@ -151,7 +150,7 @@ CHIP_ERROR ChipDeviceController::ConnectDevice(NodeId remoteDeviceId, IPAddress
     err = mSessionManager->Init(mLocalDeviceId, mInetLayer, Transport::UdpListenParameters().SetAddressType(deviceAddr.Type()));
     SuccessOrExit(err);
 
-    mSessionManager->SetDelegate(&mCallback);
+    mSessionManager->SetDelegate(this);
```

Comment:
supernit for future PRs: if this is a callback, how about starting to name things 'setCallbacks' or 'setCallbackHandler' or similar rather than Delegate?

Reply:
This is generally following a delegate pattern, why not call it `SetDelegate`?

Reply:
I guess or the reverse and rename the classes. I am advocating for consistency. Right now we have:

```
class DLL_EXPORT SecureSessionMgrCallback
...
setDelegate
```

I am proposing to either use setSessionManagerCallbacks (and woud add a 's' to the end since this is more than a single callback) or changing the class to `SecureSessionMgrDelegate`

Reply:
I'm all for `SecureSessionMgrDelegate`.